wordpress-mobile/WordPress-iOS
# ULS Site Address: show unified view from "add self-hosted" links

Ref: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/283
Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/402

This shows the unified Site Address flow from these entry points if `unifiedSiteAddress` is enabled.
- Login Epilogue > `Connect a site`
- Post Signup Interstitial > `Add a self-hosted site`
- My Sites > + > `Add self-hosted site`

To note, all three entry points use the same Auth change to display Site Address, so they should all be the same.

To test:

Login Epilogue:
- Login.
- On the Login Epilogue, tap `Connect a site`.
- Verify the new Site Address flow is displayed.

My Sites:
- Go to My Sites.
- Tap `+ on the nav bar.
- Select `Add self-hosted site`.
- Verify the new Site Address flow is displayed.

Post SignUp Interstitial:
- Hack: to force the PSI to show, in `PostSignUpInterstitialViewController.shouldDisplay` just `return true`.
- Login.
- On the PSI, tap `Add a self-hosted site`.
- Verify the new Site Address flow is displayed.

| ![light](https://user-images.githubusercontent.com/1816888/91099372-6bca8d80-e620-11ea-9f14-ef33048c19af.png) |  ![dark](https://user-images.githubusercontent.com/1816888/91099394-72f19b80-e620-11ea-9a41-965c778d7bdd.png)|
|--------|-------|


PR submission checklist:
